src/test/java/com/example/produktapi/service/ExceptionHandlerAdviceTest.java

```diff
@@ -0,0 +1,67 @@
+package com.example.produktapi.service;
+
+import com.example.produktapi.exception.BadRequestException;
+import com.example.produktapi.exception.EntityNotFoundException;
+import com.example.produktapi.model.Product;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ExceptionHandlerAdviceTest {
+
+    private ExceptionHandlerAdvice exceptionHandlerAdvice;
+
+    @BeforeEach
+    public void setup() {
+        exceptionHandlerAdvice = new ExceptionHandlerAdvice();
+    }
+
+    @Test
+    @DisplayName("unitTestofHandleEntityNotFoundException")
```

Comment:
Läsbarheten av detta `DisplayName` skulle kunna förbättras. Syftet med DisplayName är att det ska vara lättare att förstå testets syfte. Således föreslår jag att ändra det till:
```suggestion
    @DisplayName("handleException (EntityNotFoundException)")
```

I denna föreslagna ändring så framkommer det tydligt vilken funktion som testas samt vad den testas mot.

Reply:
Först vill jag börja med att tacka dig Jamie för att du tar dig tid att gå igenom så pass noggrant som du har gjort. Jag har mycket lite erfarenhet av att skriva kod och all feedback är guld värd.

Har dålig koll på best practice för hur man namnger tester. Ditt förslag låter vettigt.